generator/sbpg/targets/python.py

```diff
@@ -105,13 +101,20 @@ def has_real_message(l):
   return any(m.static and m.is_real_message for m in l)
 
 
+def needs_raw_docstring(m):
+  if m.desc and "\\0" in m.desc:
```

Comment:
when this occurs, embedding a raw `\0` into the docstring is nice to read in the Python, but it appears as just `0` in the Sphinx docs